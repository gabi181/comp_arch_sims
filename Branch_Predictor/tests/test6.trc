8 1 8 3 global_history global_tables using_share_mid
0x7C40C060 N 0x359F04A4
0x7C40C060 T 0x359F04A4
0x0C4EA178 T 0x71610120
0x7C40C060 T 0x359F04A4
0x0C4EA178 T 0x71610120
0x7C40C060 N 0x359F04A4
0x7C40C060 N 0x359F04A4
0x0C4EA178 N 0x71610120
0x4FB14D0C T 0x2A9D641C
0x7C40C060 N 0x359F04A4
0x4FB14D0C N 0x2A9D641C
