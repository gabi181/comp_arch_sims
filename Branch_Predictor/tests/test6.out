0x7c40c060 N 0x7c40c064
0x7c40c060 T 0x359f04a4
0xc4ea178 N 0xc4ea17c
0x7c40c060 T 0x359f04a4
0xc4ea178 T 0x71610120
0x7c40c060 T 0x359f04a4
0x7c40c060 T 0x359f04a4
0xc4ea178 T 0x71610120
0x4fb14d0c N 0x4fb14d10
0x7c40c060 T 0x359f04a4
0x4fb14d0c T 0x2a9d641c
flush_num: 7, br_num: 11, size: 317b
